### DISCUSSION TITLE
"Curently played note" highlight improvement

### DISCUSSION THREAD

**Discussion by logari84** [helge17/tuxguitar#364] [General]
Hi,

Tux guitar is an amazing application, though I think that the highlight of the currently played note could be improved. Sometimes it is really hard to see the highlighted not, I guess for people who are color-blinded it would be even harder. A better approach would be to place the currently played note inside a box (with curved corners it would look really nice) and with a pale background color. Apparently the box would contain both the tablature and the score.

I hope you find my feedback useful!

Thank you.

**Comment by guiv42** [comment]
Hi,
thanks for your feedback. As described in [contribute.md](https://github.com/helge17/tuxguitar/blob/master/doc/contribute.md), we are using "issues" and not "discussions" for feature requests, so I moved this discussion to #365
